fastlane/swift/ArgumentProcessor.swift

```diff
@@ -12,6 +12,7 @@ struct ArgumentProcessor {
     let args: [RunnerArgument]
     let currentLane: String
     let commandTimeout: Int
+    let laneParameters: [String: String]
```

Comment:
> `[String: String]`

We prefer: `[String : String]`
I think we should probably add Swift linting so this is automatic too